nixos/modules/services/databases/postgresql.nix

```diff
@@ -168,6 +168,18 @@ in
                 }
               '';
             };
+            ensureRoles = mkOption {
+              type = types.listOf (types.enum [ "superuser" "create role" "create db" "replication" "bypass urls" ]);
+              default = [ ];
+              description = lib.mdDoc ''
+                A list of roles to grant to the user. Under the hood this uses the
+                [ALTER USER syntax](https://www.postgresql.org/docs/current/sql-alteruser.html) for each role  in the list:
```

Comment:
Double space in "each role  in". (Hehe, it seems github removes double space in these comments.)